Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -253,7 +253,17 @@ void menu_backlash();
 //
 #if SHOW_MENU_ADVANCED_TEMPERATURE
 
+  #if ENABLED(MPC_EDIT_MENU)
+    #define MPC_EDIT_DEFS(N) \
+      MPC_t &c = thermalManager.temp_hotend[N].constants; \
+      TERN(MPC_INCLUDE_FAN, editable.decimal = c.ambient_xfer_coeff_fan0 + c.fan255_adjustment)
```

Comment:
Got an error here when I tried enabling MPC_EDIT_MENU:
Marlin/src/lcd/menu/menu_advanced.cpp:259:7: error: 'TERN' was not declared in this scope
  259 |       TERN(MPC_INCLUDE_FAN, editable.decimal = c.ambient_xfer_coeff_fan0 + c.fan255_adjustment)
as TERN takes 3 parameters and only 2 were provided. (Also, sorry, not familiar enough with code here to propose a proper fix.)

Reply:
@Sonicboom247: What mfconfig changes?

The changes in #24018 adjust which TERN is used in `menu_advanced.cpp`: https://github.com/MarlinFirmware/Marlin/pull/24018/files

Reply:
OOPS! looks like i clicked the commit [1d2f466](https://github.com/MarlinFirmware/Marlin/pull/24018/commits/1d2f4668ae7abbb04742949d7c08dfbb7a632a8c)

Reply:
That was an artifact of thinkyhead force-pushing changes to bugfix.